packages/cozy-pouch-link/src/startReplication.js

```diff
@@ -56,19 +62,21 @@ export const startReplication = (
     }
     let replication
     if (initialReplication && strategy !== 'toRemote') {
-      // For the first remote->local replication, we manually replicate all docs
-      // as it avoids to replicate all revs history, which can lead to
-      // performances issues
-      const docs = await replicateAllDocs(pouch, url)
-      return resolve(docs)
+      ;(async () => {
+        // For the first remote->local replication, we manually replicate all docs
+        // as it avoids to replicate all revs history, which can lead to
+        // performances issues
+        docs = await replicateAllDocs(pouch, url)
+        return resolve(docs)
+      })()
```

Comment:
nit: I'm a big fan of having "working" commit. So if the previous solution didn't work, this change should be squashed with the "first commit"